pilot/pkg/serviceregistry/kube/controller/namespacecontroller.go

```diff
@@ -146,7 +176,7 @@ func (nc *NamespaceController) insertDataForNamespace(ns string) error {
 		Namespace: ns,
 		Labels:    configMapLabel,
 	}
-	return k8s.InsertDataToConfigMap(nc.client, nc.configmapLister, meta, nc.getData())
+	return k8s.InsertDataToConfigMap(nc.client, nc.configmapLister, meta, nc.caBundleWatcher.GetCABundle())
```

Comment:
This is a fairly critical update without which workloads will not be able to communicate with istiod. Does it make sense to re-add the task item to the queue if there is a failure?

Reply:
Have already done for `insertDataForNamespace`